### ISSUE TITLE
Handling pagination for Comments. How many levels of nested comments deep?

### ISSUE THREAD

**Issue by marcaaron** [Wobbly-App/techno-syndicalism#27]
When a user lands on a post/conversation there should be a certain limit set to how many comments are shown.

How many top-level (a.k.a. direct responses to comments) should be shown?

How about nested comments? How many replies to comments and replies to those comments should be shown?

There should be a mechanism for further expanding comments in a particular thread, but a limit set on the initial render.

Thoughts?

**Comment by King-Mob** [comment]
Hmmm. I think you should be able to scroll through 2 screensworth of comments before you need to press "load more". So, if you have a post, and a comment of that post takes up half a screen, maybe you have 3 more comments of that size before you need to press read more again.

If that's too complex to work out, maybe we could just try max of 8 comments before you load more?

I think this would make sense to extend to the top level as well, when a user is viewing all the posts in a particular group. So, you see the top 8 posts, and maybe the first 2 comments, and clicking on a post takes you to that post's page.

NESTED COMMENTS

A user is on the page of a particular post.
Let's try having top level comments visible, and then perhaps the first 2 subcomments visible, and then an "expand" button.
So each layer deeper you go, you can see the first 2 comments that are deeper than that level, then you can expand to reveal more.

Typing this out, I'm starting to think we should allow preferences on this, so you can set to have comments always expanded or whatever, but I think that should come after we set a default behaviour first.

**Comment by King-Mob** [comment]
yeah I agree with that. I think I'd be inclined to add like, the first 2 lines of the post body and then a "..." for the group posts list view, but that's something for #production
